### ISSUE TITLE
Add Thesis tab to home page

### ISSUE THREAD

**Issue by carakey** [osulp/Scholars-Archive#2831]
### Descriptive summary

Add a tab to the home page, in the right set beside "Spotlight" / "Explore Collections", for "OSU Thesis Archive". This section would give quick access to major collections including GTDs, Honors College Theses, and other student works. 

Main benefits are improved discovery of OSU theses (as a primary SA mission); plus clearer distinction of the theses as an 'archival' set which in turn helps make the case for the ADA archive exception angle.  

### Expected behavior

Clicking the tab reveals text content that describes and links to search results for different types of student work. Content of the tab should be editable by non-dev repo admins as a "Content Block" on the Dashboard.

### Related work

#2223 
#2827

**Comment by lamtu1** [comment]
@simholt do you have some ideas in regard of displaying this having a list of collections and in those collections hold some of the GTDs, HCTs, etc... or should we just display some recent thesis work in the column and have a button for them to search and discover more?

**Comment by lamtu1** [comment]
`PRELIMINARY DISPLAY:` Below is the preliminary display of the theses display either in `collection format` or `work display`. For now, these are just notes to put down and will revisit once have confirmation:

This one display in `collection` layout similar to the `Explore Collection` tab

<img width="1166" height="565" alt="Image" src="https://github.com/user-attachments/assets/42876c84-542b-4ac9-ba86-02e1a761b6f7" />



This one below is similar to the one above but will have thumbnail and display and link to the `work` instead of a collection with recent theses:

<img width="597" height="195" alt="Image" src="https://github.com/user-attachments/assets/2d3a93af-5544-44e6-a0ed-0837735b46d2" />

**Comment by simholt** [comment]
Okay, can we instead make two tabs? 

- Honors College Theses Archive
- Theses & Dissertations Archive

At this point, they can goto the same URLs as the theses links on the main page.

**Comment by lamtu1** [comment]
> Okay, can we instead make two tabs?
> 
> * Honors College Theses Archive
> * Theses & Dissertations Archive
> 
> At this point, they can goto the same URLs as the theses links on the main page.

Hi @simholt, I can do two tabs, would you just like me to fetch the recent upload works for both and then have the button for them to explore more by bring them to the search result of works of GTD and HCT?

**Comment by lamtu1** [comment]
Yes that will be doable

**Comment by simholt** [comment]
A few changes -- 

- [ ] Change tabs to:
Theses & Dissertations Archive
Honors College Theses Archive
- [ ] can you limit the number of items viewable on each of those tab to 10? 
- [ ] this is looking crowded; can you move the Explore Collections tab over to the left column, after Recently Uploaded?

<img width="764" height="257" alt="Image" src="https://github.com/user-attachments/assets/9b6bf377-d501-4d9a-a101-019e89592c9d" />

**Comment by simholt** [comment]
Alright - QA pass-
